contracts/handlers/ERC721Handler.sol

```diff
@@ -123,16 +90,6 @@ contract ERC721Handler is IDepositExecute, HandlerHelpers, ERC721Safe {
         } else {
             lockERC721(tokenAddress, depositer, address(this), tokenID);
         }
-
-        _depositRecords[destinationChainID][depositNonce] = DepositRecord(
-            tokenAddress,
-            uint8(destinationChainID),
-            resourceID,
-            destinationRecipientAddress,
-            depositer,
-            tokenID,
-            metaData
```

Comment:
metaData should be emitted in an event in this handler along with the resourceId, destinationChainId and depositNonce.

Reply:
make sense